resolwe_bio/tests/files/HG19_FIGURE1.gff

```diff
@@ -0,0 +1,16 @@
+chr17	ARL4D	ARL4D	41468594	41566948		+		ARL4D
```

Comment:
Similar as above, existing files in `resolwe_bio/tests/files` usually use lower case for names and the prefix says something where this is used. Could you improve the name?